cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
To continue my one-sided conversation with myself, should the actual vocab implementations be in `cybox.common.vocabs` or `cybox.vocabs`? `VocabString` is in here because it is an actual type defined in the CybOX Common schema.